pulldown-cmark/src/scanners.rs

```diff
@@ -269,6 +269,26 @@ impl<'a> LineStart<'a> {
         }
     }
 
+    pub(crate) fn scan_spoiler_fence(&mut self) -> bool {
+        if self.scan_case_insensitive(b":::") {
```

Comment:
commonmark-hs allows you to use three or more opening delimiters, not just three. Please try to implement extensions in a compatible way: we really want to avoid inventing our own syntax wherever feasible.